tsconfig.base.json

```diff
@@ -21,6 +21,9 @@
       "@griffel/e2e-utils": ["e2e/utils/src/index.ts"],
       "@griffel/eslint-plugin": ["packages/eslint-plugin/src/index.ts"],
       "@griffel/jest-serializer": ["packages/jest-serializer/src/index.ts"],
+      "@griffel/linaria-processor": ["packages/linaria-processor/src/index.ts"],
+      "@griffel/linaria-processor/make-styles": ["packages/linaria-processor/src/MakeStylesProcessor.ts"],
+      "@griffel/linaria-processor/make-reset-styles": ["packages/linaria-processor/src/MakeResetStylesProcessor.ts"],
```

Comment:
Aliases that match export maps in `@griffel/linaria-processor`, needed for Jest.